java/src/org/openqa/selenium/grid/node/ProxyNodeWebsockets.java

```diff
@@ -49,7 +49,7 @@ public class ProxyNodeWebsockets
     implements BiFunction<String, Consumer<Message>, Optional<Consumer<Message>>> {
 
   private static final UrlTemplate CDP_TEMPLATE = new UrlTemplate("/session/{sessionId}/se/cdp");
-  private static final UrlTemplate BIDI_TEMPLATE = new UrlTemplate("/session/{sessionId}/se/bidi");
+  private static final UrlTemplate BIDI_TEMPLATE = new UrlTemplate("/session/{sessionId}");
```

Comment:
I think we can keep the endpoint, just set the URL we are creating in `LocalNode` in the `webSocketUrl` capability.